.gitignore

```diff
@@ -0,0 +1,8 @@
+__pycache__
```

Comment:
pyc binaries are coming in, add in https://github.com/github/gitignore/blob/main/Python.gitignore here?